static/styles/rendered_markdown.scss

```diff
@@ -1,4 +1,8 @@
 .rendered_markdown {
+    /* To make message horizontally scrollable
+    if rendered markdown message is overflowing. */
+    overflow-x: auto !important;
```

Comment:
Do we want this on the whole message object?  I'd have thought we'd just want to set this on table elements inside it.

(I could imagine this change breaking various things).

Reply:
ok, I will make it local to the scope.

Reply:
I now added the styles inside the table elements and it's working fine now.